pkg/service/mocktest/mocktest.go

```diff
@@ -51,7 +51,7 @@ func (s *mockTester) MockTest(path string, pid uint32, mockName string) {
 	}
 
 	// start the proxy
-	ps := proxy.BootProxy(s.logger, proxy.Option{}, "", "", pid, "", []uint{}, loadedHooks, ctx)
+	ps := proxy.BootProxy(s.logger, proxyPort, proxy.Option{}, "", "", pid, "", []uint{}, loadedHooks, ctx)
```

Comment:
You could use the proxy "Option" field, i.e. Port only, rather than sending proxyPort explicitly in all command functions.